bin/akula.rs

```diff
@@ -221,29 +225,62 @@ fn main() -> anyhow::Result<()> {
                                 .unwrap();
 
                             let mut api = Methods::new();
-                            api.merge(
-                                EthApiServerImpl {
-                                    db: db.clone(),
-                                    call_gas_limit: 100_000_000,
-                                }
-                                .into_rpc(),
-                            )
-                            .unwrap();
-                            api.merge(NetApiServerImpl { network_id }.into_rpc())
-                                .unwrap();
-                            api.merge(ErigonApiServerImpl { db: db.clone() }.into_rpc())
-                                .unwrap();
-                            api.merge(OtterscanApiServerImpl { db: db.clone() }.into_rpc())
-                                .unwrap();
-                            api.merge(
-                                TraceApiServerImpl {
-                                    db,
-                                    call_gas_limit: 100_000_000,
+
+                            for api_option in opt
+                                .enable_api
+                                .iter()
+                                .map(|o| o.to_lowercase())
+                                .collect::<Vec<String>>()
+                            {
+                                match api_option.as_str() {
+                                    "eth" => {
```

Comment:
Test if hashset contains id for each API, instead of enabling API for each id

Reply:
Ok, sounds good. I will update this shortly.